scripts/mbedtls_framework/c_parsing_helper.py

```diff
@@ -60,18 +62,37 @@ def __init__(self, #pylint: disable=too-many-arguments
                  qualifiers: Iterable[str],
                  return_type: str,
                  name: str,
-                 arguments: List[str]) -> None:
+                 arguments: List[str],
+                 doc: str = "") -> None:
+
         self.filename = filename
         self.line_number = line_number
         self.qualifiers = frozenset(qualifiers)
         self.return_type = return_type
         self.name = name
         self.arguments = [ArgumentInfo(arg) for arg in arguments]
+        self.doc = doc
 
     def returns_void(self) -> bool:
         """Whether the function returns void."""
         return bool(self.VOID_RE.search(self.return_type))
 
+    def __str__(self) -> str:
+        _str_args = [str(a) for a in self.arguments]
```

Comment:
Minor: this shouldn't be an underscore-prefixed variable name by any of the conventions I know (class members that child classes should not access, and potentially unused variables).